lib/internal/bootstrap/loaders.js

```diff
@@ -178,17 +185,31 @@ let internalBinding;
   };
 }
 
+const getOwn = (target, property, receiver) => {
+  return ObjectPrototypeHasOwnProperty(target, property) ?
+    ReflectGet(target, property, receiver) :
+    undefined;
+};
+
 const loaderId = 'internal/bootstrap/loaders';
 const {
   builtinIds,
   compileFunction,
 } = internalBinding('builtins');
 
-const getOwn = (target, property, receiver) => {
-  return ObjectPrototypeHasOwnProperty(target, property) ?
-    ReflectGet(target, property, receiver) :
-    undefined;
-};
+const publicBuiltinIds = builtinIds
+  .filter((id) =>
+    !StringPrototypeStartsWith(id, 'internal/') &&
+      !experimentalModuleList.has(id),
+  );
+// Do not expose the loaders to user land even with --expose-internals.
```

Comment:
Why not?

Reply:
I don't remember the details, but that was specifically done https://github.com/nodejs/node/pull/19112#issuecomment-370239518 and there is a test for it in `test/es-module/test-loaders-hidden-from-users.js` (why was this moved to `es-module`?!), this is just a refactoring that preserves the behavior.

Reply:
It was probably moved into `es-module` by mistake because generally everything related to loaders is in `es-module`. This is just about the only reference I’ve seen in a long time to CommonJS “loaders.” If you want to move it somewhere else, we really should have `test/module` for CommonJS stuff; it doesn’t belong in `test/parallel` (nothing does, really, but that’s its own PR). For the purposes of this PR I’d just leave it where it is.

Getting back to the original note, I was just hoping that we could explain this a bit more in the code comment; but if you don’t remember the details that’s fine, we can just leave it as is.